services/pub/pub.tester.js

```diff
@@ -20,7 +20,7 @@ t.create('package pre-release version')
 
 t.create('package not found').get('/v/does-not-exist.json').expectBadge({
   label: 'pub',
-  message: 'not found',
+  message: 'invalid',
```

Comment:
Thanks for picking up these service test failures but this is not quite the right fix. I've submitted a PR #7921 for the service test failures. Lets get that one merged and then circle back, rebase these 3 and account for any changes to the error handling and then revisit the reviews.

Reply:
Thank you for the PR to fix the pub service test.